core/modules/entity/entity.module

```diff
@@ -185,12 +185,11 @@ function entity_create_stub_entity($entity_type, $ids) {
  * database access if loaded again during the same page request.
  *
  * The actual loading is done through a class that has to implement the
- * EntityControllerInterface interface. By default,
- * DefaultEntityController is used. Entity types can specify that a
- * different class should be used by setting the 'controller class' key in
- * hook_entity_info(). These classes can either implement the
- * EntityControllerInterface interface, or, most commonly, extend the
- * DefaultEntityController class. See node_entity_info() and the
+ * EntityControllerInterface interface. By default,DefaultEntityController is
```

Comment:
Just checkout the same branch again, make the fix then:
```
git commit --amend
git push origin [branch_name] -f
```

Yes, GitHub supports force-updating PRs (even in a way encourages it). It's what drives Linus Torvalds mad about GitHub.